Inc/ST-LIB_LOW/Sensors/EncoderSensor/EncoderSensor.hpp

```diff
@@ -0,0 +1,32 @@
+
+/*
+ * AnalogSensor.hpp
+ *
+ *  Created on: Nov 9, 2022
+ *      Author: ricardo
+ */
+
+#pragma once
+#include "ST-LIB.hpp"
+#include "ST-LIB_LOW/Sensors/Sensor/Sensor.hpp"
+#define counter_distance 0.025
+#define clock_frequency 300000000
```

Comment:
```suggestion
#define CLOCK_FREQUENCY 300000000
```
constantes en mayuscula